src/ir/comp.rs

```diff
@@ -1473,8 +1473,27 @@ impl DotAttributes for CompInfo {
 impl IsOpaque for CompInfo {
     type Extra = ();
 
-    fn is_opaque(&self, _: &BindgenContext, _: &()) -> bool {
-        self.has_non_type_template_params
+    fn is_opaque(&self, ctx: &BindgenContext, _: &()) -> bool {
+        //Early return to avoid extra computation
+        if self.has_non_type_template_params {
+            return true
+        }
+
+        let has_large_bitfield = self.fields().iter().any(|f| match *f {
```

Comment:
Nitpick: rather than assigning the bool expression to a variable and then returning the variable, we should return the expression directly.